samples/OpenIdConnectSample/Startup.cs

```diff
@@ -102,7 +98,6 @@ await context.Authentication.SignOutAsync(OpenIdConnectDefaults.AuthenticationSc
                     {
                         RedirectUri = "/signedout"
                     });
-                    await context.Authentication.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
```

Comment:
I'm fine reverting this change, since it's what any client app should do. But note that removing the local cookie before redirecting the user agent to the identity provider doesn't allow debugging the `sid` check.

Since this sample is more a sandbox than a real sample, I thought it was better to remove it.